e2e/testcafe-devextreme/tests/dataGrid/sticky/common/withBandColumns.ts

```diff
@@ -0,0 +1,52 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import DataGrid from 'devextreme-testcafe-models/dataGrid';
+import { safeSizeTest } from '../../../../helpers/safeSizeTest';
+import { createWidget } from '../../../../helpers/createWidget';
+import url from '../../../../helpers/getPageUrl';
+
+const DATA_GRID_SELECTOR = '#container';
+
+fixture.disablePageReloads`Band sticky columns`
+  .page(url(__dirname, '../../../container.html'));
+
+[false, true].forEach((rtlEnabled) => {
+  // T1279722
+  safeSizeTest(`Headers and filter row should display correctly after scrolling to the max right position when there is a grouped column (rtl=${rtlEnabled})`, async (t) => {
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+    const dataGrid = new DataGrid(DATA_GRID_SELECTOR);
+
+    await t.expect(dataGrid.isReady()).ok();
+
+    await dataGrid.scrollTo(t, { x: rtlEnabled ? 0 : 10000 });
+    await takeScreenshot(`T1279722_band_sticky_columns-headers_with_filter_row_and_grouped_column_(rtl=${rtlEnabled}).png.png`, dataGrid.element);
```

Comment:
Let's remove duplication of file extension in etalon name
```suggestion
    await takeScreenshot(`T1279722_band_sticky_columns-headers_with_filter_row_and_grouped_column_(rtl=${rtlEnabled}).png`, dataGrid.element);
```